sync_test.go

```diff
@@ -43,6 +61,16 @@ func TestFsSyncer_Sync(t *testing.T) {
 		"it should keep the symlink to a relative path": {
 			fixtureSrc: "src/relative-symlink",
 		},
+		"it should replace a file with the same content but not the same mtime": {
+			fixtureSrc:      "src/file",
+			fixtureDst:      "dst/cp-file",
+			expectedChanges: []string{"a"},
+		},
+		"it should replace a file with the same size but not the same mtime": {
+			fixtureSrc:      "src/file",
+			fixtureDst:      "dst/same-size",
+			expectedChanges: []string{"a"},
+		},
```

Comment:
These tests are impossible using on the CI as a `git clone` does not keep the modification time of the files. We would love to have `src/file` and `dst/cp-file` with two different modification time. But after a `git clone` on the CI, both files have the same modification time. It's the time of the `git clone`.